src/linear_algebra/char_poly/coeff.lean

```diff
@@ -0,0 +1,191 @@
+/-
+Copyright (c) 2020 Aaron Anderson, Jalex Stark. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Aaron Anderson, Jalex Stark.
+-/
+
+import linear_algebra.char_poly
+import linear_algebra.matrix
+import ring_theory.polynomial.basic
+import algebra.polynomial.big_operators
+
+/-!
+# Characteristic polynomials
+
+We give methods for computing coefficients of the characteristic polynomial.
+
+## Main definitions
+
+- `degree_char_poly_eq_dim`
+- `det_from_char_poly`
+- `trace_from_char_poly`
+
+## Implementation details
+
+-/
+
+noncomputable theory
+
+universes u v w z
+
+open polynomial matrix
+open_locale big_operators
+
+variables {R : Type u} [comm_ring R]
+variables {n : Type v} [fintype n] [decidable_eq n]
+variables {α : Type v} [decidable_eq α]
+
+open finset
+open polynomial
+
+section fixed_points
+-- TODO: move this section to be with other combinatorial facts about the symmetric group
+lemma gt_one_nonfixed_point_of_nonrefl {σ : equiv.perm n} (h : σ ≠ equiv.refl n) :
+1 < (filter (λ (x : n), ¬ σ x = x) univ).card :=
```

Comment:
I think `_nonrefl` is nonstandard, and probably should be `_ne_refl`. `nonfixed` sounds a bit strange, but I don't have a suggestion.

Reply:
Also, `one_lt` would be more precisely following the naming conventions...